libs/core/kiln_ai/datamodel/extraction.py

```
@@ -78,15 +109,88 @@ class ExtractorConfig(KilnBaseModel):
+    KilnParentedModel, KilnParentModel, parent_of={"extractions": Extraction}
+):
+    name: str = NAME_FIELD
+
+    description: str = Field(description="A description for the file")
+
+    original_file: FileInfo = Field(description="The original file")
+
+    # TODO: move {mime_type:kind} mapping out of GeminiExtractor and into here
+    #   - will also need to have models specify which mimetypes they support
+    kind: Kind = Field(
+        description="The kind of document. The kind is a broad family of filetypes that can be handled in a similar way"
```

Pylint (3.3.7)

[convention] Line too long (120/100)

(C0301)